code/obj/machinery/firealarm.dm

```diff
@@ -52,27 +53,27 @@
 
 /obj/machinery/firealarm/proc/toggleinput(var/datum/mechanicsMessage/inp)
 	if(src.icon_state == "fire0")
-		alarm()
+		alarm(1)
 	else
 		reset()
 	return
 
 /obj/machinery/firealarm/temperature_expose(datum/gas_mixture/air, temperature, volume)
 	if(src.detecting)
 		if(temperature > T0C+200)
-			src.alarm()			// added check of detector status here
+			src.alarm(1)			// added check of detector status here
 	return
 
 /obj/machinery/firealarm/attack_ai(mob/user as mob)
 	return src.attack_hand(user)
 
 /obj/machinery/firealarm/bullet_act(BLAH)
-	return src.alarm()
+	return src.alarm(5)
 
 /obj/machinery/firealarm/emp_act()
 	..()
 	if(prob(50))
-		src.alarm()
+		src.alarm(1)
```

Comment:
Magic numbers bad. I suggest putting some defines in the _std folder. Maybe like `#DEFINE ALARM_FIRE 1` etc